tests/foreman/ui/test_oscap_tailoringfiles.py

```diff
@@ -45,6 +53,15 @@ def setUpClass(cls):
         cls.loc_name = loc.name
         org = entities.Organization(name=gen_string('alpha')).create()
         cls.org_name = org.name
+        scap_path = settings.oscap.content_path
+        cls.content_path = get_data_file(scap_path)
+        proxy = entities.SmartProxy().search(
+            query={
+                u'search': u'name={0}'.format(
+                    settings.server.hostname)
+            }
+        )[0]
+        proxy.organization = [org]
```

Comment:
why we need this assignment if we are not updating the proxy?
is this not dangerous for other tests, perhaps we have to append the org to the already existent proxy organizations

Reply:
Note: we are updating the default proxy, this should affect all tests

Reply:
This appends the org to the capsule , doesn't remove the existing org's. But , yes it could be implemented in better way , looking